test/cartesian/ReferenceLine/ReferenceLine.spec.tsx

```diff
@@ -296,4 +296,62 @@ describe('<ReferenceLine />', () => {
       y2: -102.22222222222223,
     });
   });
+
+  test('throws when passed in invalid xAxisId', () => {
+    const viewBox: CartesianViewBox = { x: 1, y: 2 };
+    expect(() => {
+      render(
+        <BarChart
+          width={1100}
+          height={250}
+          barGap={2}
+          barSize={6}
+          data={data}
+          margin={{ top: 20, right: 60, bottom: 0, left: 20 }}
+        >
+          <XAxis dataKey="name" />
+          <YAxis tickCount={7} />
+          <Bar dataKey="uv" />
```

Comment:
Do we need the bar for this test? Ideally the test should be as simple as possible.

Reply:
Done